### ISSUE TITLE
Waterfox version incorrect? No menu item to disable automatic updates (automatic breakages)

### ISSUE THREAD

**Issue by teotikalki** [BrowserWorks/waterfox#142]
Help -> About Firefox says 45.8.0
Add-ons Manager says "Vertical Tabs Reloaded is incompatible with Firefox 45.8.0" (Also FEBE, GNOME Shell Integration, Privacy Badger).

Synaptic tells me that package waterfox has installed verstion 52.0.2. I had actually done a snapshot-rollback yesterday to undo this and get back browser functionality (I have lots of tabs open even after a few minutes of a new sessions; Tree Style Tabs has been my go-to for years, literally life-changing; I've used Vertical Tabs Reloaded with Australis (Palemoon is my main browser, or was until I set up Australis beautifully... and Waterfox replaced Firefox for me as soon as Firefox stated that my setup would no longer work after v57). I tried to lock version in synaptic to keep this from happening again, but no luck. I dislike that I can't seem to find a setting in the menu (googled instructions state that it should be under Preferences -> Advanced -> Update; I found that I can change the property in about:config which seems like a next move...)
I strongly dislike the idea of involuntary automatic BREAKAGES. I don't want to 'update' the browser unless I know that it won't cost me my browser environment!

Edit: I am using Debian Stretch with Waterfox installed from the hawkeye116477 repository.

**Comment by teotikalki** [comment]
Additional note: I am using Debian Stretch GNOME. I wish that the repository didn't try to force me to install KDE integration!

**Comment by hawkeye116477** [comment]
If you installed Waterfox from my repo, then you have manual updates together with system updates. KDE integration features are only optional to install.  If you don't want to update Waterfox together with other system updates, then remove repo and download and install latest package from here -> https://github.com/hawkeye116477/waterfox-deb/tree/master/pool/main/w/waterfox .

Update: On 53.0 menu item to disable auto-updates should be back again (only on standard edition, not KDE), but I removed updater and set auto-updates disabled as default, because updates are through repository, so this is not needed. Previously updater also was removed, but this updater and setting on menu has nothing to do with Synaptic.
